.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
-xcuserdata
\ No newline at end of file
+xcuserdata
+xcuserstate
```

Comment:
```suggestion
*.xcuserstate
```

Reply:
@GabePires updated.
what do those characters mean? Mainly how will I know how/when to use those

Reply:
I think it's like a "wildcard"
So `*.` means "anything"
Basically, any file that ends with xcuserstate should be ignored.... which im not sure why it's not lol